src/psbt.h

```diff
@@ -50,6 +50,9 @@ static constexpr uint8_t PSBT_IN_TAP_LEAF_SCRIPT = 0x15;
 static constexpr uint8_t PSBT_IN_TAP_BIP32_DERIVATION = 0x16;
 static constexpr uint8_t PSBT_IN_TAP_INTERNAL_KEY = 0x17;
 static constexpr uint8_t PSBT_IN_TAP_MERKLE_ROOT = 0x18;
+static constexpr uint8_t PSBT_IN_MUSIG2_PARTICIPANT_PUBKEYS = 0x19;
+static constexpr uint8_t PSBT_IN_MUSIG2_PUB_NONCE = 0x1a;
+static constexpr uint8_t PSBT_IN_MUSIG2_PARTIAL_SIG = 0x1b;
```

Comment:
These three constants are `0x1a` , `0x1b`, `0x1c` in the final version of [BIP-373](https://github.com/bitcoin/bips/blob/master/bip-0373.mediawiki).

Reply:
Good catch! Fixed.